src/language-server/feature-finder.ts

```diff
@@ -105,15 +153,33 @@ export default class FeatureFinder {
   async getAstAtPosition(document: Document, position: SourcePosition):
       Promise<AstLocation|undefined> {
     const parsedDocument = document.parsedDocument;
-    if (!(parsedDocument instanceof ParsedHtmlDocument)) {
-      return;
+    if (parsedDocument instanceof ParsedHtmlDocument) {
+      const node = getHtmlAstLocationForPosition(parsedDocument, position);
+      if (node && node.kind === 'styleTagContents') {
+        const cssDocuments = document.getFeatures({kind: 'css-document'});
+        if (node.textNode) {
```

Comment:
this condition check can be moved up to avoid querying for cssDocuments, e.g.
```js
if (node && node.kind === 'styleTagContents' && node.textNode) {
  const cssDocuments = document.getFeatures({kind: 'css-document'});
  const styleElement = node.textNode.parentNode;
  ...
```